jep/229/README.adoc

```diff
@@ -351,6 +367,14 @@ implements basic aspects of this proposal from the developer side.
 link:https://github.com/jenkins-infra/repository-permissions-updater/pull/1747[repository-permissions-updater #1747]
 implements changes to infrastructure.
 
+link:https://github.com/jenkinsci/plugin-pom/pull/375[plugin-pom #375] and link:https://github.com/jenkinsci/pom/pull/147[pom #147]
+add a POM profile to block MRP in CD mode.
```

Comment:
It is unclear what this is for. Prevent accidental misuse?

Reply:
Yes, prevents you from running `mvn release:prepare release:perform`, which would produce crazy results. If you opt into `changelist.format`, you agree not to use MRP.